ModelUploader/Program.cs

```diff
@@ -132,6 +132,8 @@ private static bool UploadModel(string file)
             string dtdl = r.ReadToEnd();
             r.Close();
 
+            object dtdlObj = JsonSerializer.Deserialize<object>(dtdl);
```

Comment:
Why deserialize into a base object (that then needs to be cast to JsonElement -- could that fail?) rather than parse directly into JsonDocument?

Reply:
That's a good enhancement that could be addressed in the next iteration. Thanks.